internal/service/store/idx_keys.go

```diff
@@ -0,0 +1,36 @@
+// Copyright 2022 MIMIRO AS
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package store
+
+import (
+	"encoding/binary"
+
+	"github.com/mimiro-io/datahub/internal/server"
+)
+
+func SeekChanges(datasetID uint32, since uint64) []byte {
```

Comment:
Do these need to be public methods?Also seem to be more Builders than Seek methods, so not sure about naming.